Resources/Private/Templates/Backend/Administration/ShowDefinition/Process.html

```diff
@@ -7,6 +7,18 @@
     <f:render partial="Backend/Administration/HeaderButtons" section="Buttons" arguments="{_all}" />
 </f:section>
 
+<f:section name="HeaderButtonsRight">
+    <f:alias map="{label: '{nz:t(key: \'Backend/Module/Administration/Definition/Show:refresh\')}'}">
```

Comment:
Does this key exists already?

Reply:
[Yes it does!](https://github.com/CuyZ/NotiZ/blob/798e82e90d89fdc28a0dd9cc1c7af636aedbd3b9/Resources/Private/Language/Backend/Module/Administration/Definition/Show.xlf#L11-L13)